### PR TITLE
SIG Governance: Split Chair & TL roles

### PR DESCRIPTION
**Pull request by mrbobbytables**
This PR removes the Tech Lead role from automatically rolling up into the Chair role and explicitly defines the TL responsibilities. It incorporates some language from #7095 to further clarify responsibilities around KEP stewardship.

TL;DR of changes:
- Both Chair and TL suggested # of leads is now 2+
- Removed quarterly reporting requirement to k-dev from Chair role (this hasn't really ever been done)
- Both Chair and TL are responsible for maintaining KEP metadata / state (but may delegate responsibility)
- TL's are responsible for reviewing and approving KEPs (but may delegate for individual KEPs)

Note - there are a number of other things in the governance doc that are not inline with how we as a project function day to day (namely around subproject creation requiring a KEP). Those changes along with general formatting & readability improvements I would like to do a separate PR to keep this focused on the splitting of the roles.

/hold for comment
/committee steering

### CODE REVIEW

File: committee-steering/governance/sig-governance.md
Comment (SergeyKanzhelev) [inline committee-steering/governance/sig-governance.md:42]:
```suggestion
of a Chair, Tech Lead, or Subproject Owner role. Leads may, and frequently do
```

oxford comma

File: committee-steering/governance/sig-governance.md
Comment (SergeyKanzhelev) [inline committee-steering/governance/sig-governance.md:119]: is it not optional any longer? nice

File: committee-steering/governance/sig-governance.md
Comment (SergeyKanzhelev) [inline committee-steering/governance/sig-governance.md:130]:
```suggestion
  another Lead or contributor to fulfill these responsibilities for individual proposals
```

maybe approver or area owner

File: committee-steering/governance/sig-governance.md
Comment (SergeyKanzhelev) [inline committee-steering/governance/sig-governance.md:117]: do you also want to extend the document contributors/chairs-and-techleads/technical-lead.md as in https://github.com/kubernetes/community/pull/7095/files?

**Review by SergeyKanzhelev — COMMENTED** [review]
lgtm

with a few notes

File: committee-steering/governance/sig-governance.md
Comment (mrbobbytables) [reply committee-steering/governance/sig-governance.md:119]: Correct - with it no longer rolling up to chair, it needs to be explicitly defined (even if folks are serving both roles). We'll realistically have a good chunk of people acting as both chair and tl for awhile, but hopefully more folks will split responsibilities.

File: committee-steering/governance/sig-governance.md
Comment (mrbobbytables) [reply committee-steering/governance/sig-governance.md]: updated all language to be "other contributors".

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

File: committee-steering/governance/sig-governance.md
Comment (palnabarun) [inline committee-steering/governance/sig-governance.md:93]: > other contributors

💯 This is a great way to grow contributors into leadership roles.

**Review by palnabarun — APPROVED** [review]
/lgtm
/approve

File: committee-steering/governance/sig-governance.md
Comment (mrbobbytables) [reply committee-steering/governance/sig-governance.md:117]: We can update that one out of band 👍 it's more general best practices.

File: committee-steering/governance/sig-governance.md
Comment (cblecker) [inline committee-steering/governance/sig-governance.md:124]:
```suggestion
- *MUST* Resolve cross-subproject and cross-SIG technical issues and decisions
  or delegate to another Lead as needed
```

**Review by cblecker — COMMENTED** [review]
LGTM, one comment to clarify

File: committee-steering/governance/sig-governance.md
Comment (mrbobbytables) [reply committee-steering/governance/sig-governance.md:124]: 👍 will update, left over form previous language.

File: committee-steering/governance/sig-governance.md
Comment (danwinship) [inline committee-steering/governance/sig-governance.md:133]: That document still describes the Tech Lead role as optional

Comment (mrbobbytables) [reply committee-steering/governance/sig-governance.md:133]: that doc is a general 'what sort of stuff does a tl do' and not part of governance docs, it can be updated anytime out of band 👍

File: committee-steering/governance/sig-governance.md
Comment (msau42) [inline committee-steering/governance/sig-governance.md:86]: In addition to what?

Comment (mrbobbytables) [reply committee-steering/governance/sig-governance.md:86]: It's poor formatting from before :|

```
- Number: 2-3
- Membership tracked in [sigs.yaml]  
  - If no tech lead role is present, Chair assumes responsibilities from [#tech-lead] section.

  In addition, run operations and processes governing the SIG:
```

It's something I plan to clean up in my next PR.

**Review by saschagrunert — APPROVED** [review]

**Comment by mrbobbytables** [comment]
thanks for all the comments, no blockers for merging :)

/hold cancel

**Pull request merged**
